### DISCUSSION TITLE
Unable to attach to CoreCLR (Operation not supported) when launching WinUI 3 Unpackaged project.

### DISCUSSION THREAD

**Discussion by LuvW-Q** [microsoft/WindowsAppSDK#2743] [Q&A]
![屏幕截图 2022-07-16 101906](https://user-images.githubusercontent.com/45089374/179335819-dbdb135e-2c72-4501-9952-8b8feb072d01.png)

As shown in the picture, when I run a WinUI3 Unpackaged project, it immediately shows this error and stops debugging after clicking OK.It shows "**_Unable to attach to CoreCLR. Unknown error: 0x80004005_**".

I looked up articles on the web, but all the solutions given were for "access denied" errors. I also tried **resetting my computer**, but that didn't help either! What should I do?

**Comment by DHancock** [comment]
I had the same error after updating a project to WindowsAppSDK 1.1.2. The fix for me was to downgrade the VS nuget package to an earliear version where the associated runtime version had previously been installed on my dev machine (1.1.1 in my case). The 1.1.2 runtime installer has issues apparently.

**Reply by LuvW-Q** [reply]
Thank you. Do you mean downgrading Microsoft.WindowsAppSDK to version 1.1.1? But after I downgraded, the problem is still not solved?
